hr-data-generator/src/main/java/uk/gov/gchq/palisade/example/hrdatagenerator/types/PhoneNumber.java

```diff
@@ -22,13 +22,16 @@
 import java.util.StringJoiner;
 
 public class PhoneNumber {
+    private static final int MAX_EXTRA_CONTACTS = 3;
+    private static final int PHONE_NUMBER_LENGTH = 10; // excluding leading zero
+
     private String type; // is this a home number, work number, mobile number ...
-    private String phoneNumber;
+    private String number;
```

Comment:
variable should be kept as "phoneNumber"

Reply:
Getters kept the same, but sonarqube complains that phoneNumber var shadows PhoneNumber clas